src/field-builder.ts

```diff
@@ -1,9 +1,16 @@
-import { FieldKind, ObjectRef, Resolver, RootFieldBuilder, SchemaTypes } from '@pothos/core';
+import { FieldKind, ObjectRef, RootFieldBuilder, SchemaTypes } from '@pothos/core';
 import { ConnectionShape } from '@pothos/plugin-relay';
 import { Cause, Context, Effect, Exit, Function, Layer, Option, pipe } from 'effect';
 import { GraphQLResolveInfo } from 'graphql';
 
 import type * as EffectPluginTypes from './types';
+import { queryFromInfo } from '@pothos/plugin-prisma';
```

Comment:
This will require all users of this plugin to have Prisma installed.  I'd probably put everything related to Prisma in a sub export so you import something like prisma-effects-plugin/prisma

Reply:
I don't have knowledge enought and I have a question about it. is sub export not requires install prisma plugin to users?

I think as a result, resolve sub export like `pothos-plugin-effect/prisma` should be try import `@pothos/plugin-prisma`.

Reply:
btw, by 3d7b25faf6d4449b3604e650c551ba4e3ddfe287 commit, now I don't need to import queryFromInfo from prisma plugin package. 😅